### PR TITLE
th-373: Fix order locations

### PR DESCRIPTION
**Pull request by MaxHopaytsa**
- Changed the column type for coordinates to jsonb
- Fixed a bug with the map display on the order screen

### CODE REVIEW

File: backend/src/libs/packages/database/schema/tables-schema.ts [hunk 1]
```diff
@@ -14,12 +15,13 @@ import {
...
 const orderStatus = pgEnum('order_status', ORDER_STATUSES);
+
 const orders = pgTable('orders', {
   id: serial('id').primaryKey(),
-  price: integer('price').notNull(),
+  price: real('price').notNull(),
   scheduledTime: timestamp('scheduled_time', { mode: 'string' }).notNull(),
-  startPoint: varchar('start_point').notNull(),
-  endPoint: varchar('end_point').notNull(),
+  startPoint: jsonb('start_point').$type<google.maps.LatLngLiteral>().notNull(),
```
Comment (Druxight) [inline backend/src/libs/packages/database/schema/tables-schema.ts]: create our own type called Coordinates

**Review by Druxight — CHANGES_REQUESTED** [review]

File: frontend/src/libs/packages/map/map-connector.package.ts [hunk 2]
```diff
@@ -30,13 +31,16 @@ class MapConnector implements IMapConnector {
       const loader = new Loader({
         apiKey,
       });
-      const [GeocodingLibrary, RoutesLibrary] = await Promise.all([
-        loader.importLibrary('geocoding'),
-        loader.importLibrary('routes'),
-      ]);
+      const [GeocodingLibrary, RoutesLibrary, AutocompleLibrary] =
```
Comment (h0wter) [inline frontend/src/libs/packages/map/map-connector.package.ts:34]: I think PlacesLibrary is more appropriate here.
```suggestion
      const [GeocodingLibrary, RoutesLibrary, PlacesLibrary] =
```

File: frontend/src/libs/packages/map/map.package.ts [hunk 3]
```diff
@@ -36,6 +37,8 @@ class MapService implements IMapService {
 
   private routes!: google.maps.DistanceMatrixService;
 
+  private autocomplete!: google.maps.places.AutocompleteService;
```
Comment (h0wter) [inline frontend/src/libs/packages/map/map.package.ts:40]: Also here, we may need to use something else from the library, and here it's just Autocomplete.
```suggestion
  private places!: google.maps.places.placesLibrary;
```

File: frontend/src/pages/order-status/order-status.tsx [hunk 4]
```diff
@@ -57,13 +57,13 @@ const OrderStatusPage: React.FC = () => {
...
   const truckLocation = useAppSelector(selectTruckLocation);
-  const mapReference = useRef<HTMLDivElement>(null);
 
   useAppMap({
     center: truckLocation ?? DEFAULT_CENTER,
-    destination: order ? jsonToLatLngLiteral(order.startPoint) : null,
+    destination: order ? order.startPoint : null,
     className: styles.map,
     mapReference: mapReference,
+    onMapLoad: () => true,
```
Comment (h0wter) [inline frontend/src/pages/order-status/order-status.tsx:65]: What's this here for?

**Review by h0wter — CHANGES_REQUESTED** [review]

**Review by h0wter — APPROVED** [review]

**Review by DmytroVoronkov — APPROVED** [review]

**Review by PavloSukhinin — APPROVED** [review]

**Review by Druxight — APPROVED** [review]

**Pull request merged**